lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOMembershipInvtnSubmissionDAOImplTest.java

```diff
@@ -250,4 +250,20 @@ public void testDeleteByTeamAndUser() throws Exception {
 		assertEquals(0, membershipInvtnSubmissionDAO.getOpenByTeamAndUserCount(teamId, pgLong, expiresOn.getTime()-1000L));
 	}
 	
+	@Test
```

Comment:
Add comment to mention the jira ticket that you are fixing. 
/*
 * PLFM-4479
 */